target/linux/qualcommax/image/ipq807x.mk

```diff
@@ -61,6 +61,20 @@ define Device/buffalo_wxr-5950ax12
 endef
 TARGET_DEVICES += buffalo_wxr-5950ax12
 
+define Device/cmcc_rm2-6
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := CMCC
+	DEVICE_MODEL := RM2-6
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_DTS_CONFIG := config@ac02
+	SOC := ipq8070
+	IMAGE/factory.ubi := append-ubi | qsdk-ipq-factory-nand
```

Comment:
Please dont override the default UBI image, but rather add `factory.bin` for the QSDK compatible image